### PR TITLE
Moves individual learner engagement to a top-level section

### PR DESCRIPTION
**Pull request by lamagnifica**
- [x] @thallada 
- [x] @dsjen 
- [x] @stroilova 
- [x] @mulby 

HTML builds without warning or error.

### CODE REVIEW

**Comment by thallada** [comment]
:+1:
For my PR, would we want to link to `learners/index.html` or `learners/Individual_Learners.html` from the Learners section?

Edit: ah, just saw your comment on my PR.

**Comment by dsjen** [comment]
👍

**Comment by mulby** [comment]
:+1:

File: docs/en_us/dashboard/source/learners/index.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.. _Learner Index:
+
+################################################
+Individual Learners
+################################################
+
+.. note:: This feature is not yet supported. EdX is currently testing the
+ individual learner activity feature.
```
Comment (stroilova) [inline docs/en_us/dashboard/source/learners/index.rst:8]: Perhaps:
EdX is currently testing the new Learners lens, which would give insight into activity of individual students.

**Comment by stroilova** [comment]
LGTM, Minor suggestion inline. 
 👍

File: docs/en_us/dashboard/source/learners/index.rst (hunk 1 above)
Comment (lamagnifica) [inline docs/en_us/dashboard/source/learners/index.rst:8]: This note has been the subject of a lot of comment and revision already. My tendency is to leave it alone now (unless you find it misleading or offensive).

Comment (stroilova) [inline docs/en_us/dashboard/source/learners/index.rst:8]: Sounds good! It is clear as is.

**Pull request merged**